app/assets/javascripts/sync.coffee

```diff
@@ -154,6 +171,60 @@ class Sync.Pusher.Subscription
   cancel: ->
     @client.unsubscribe(@channel) if @client.channel(@channel)?
 
+class Sync.Stomp extends Sync.Adapter
+
+  subscriptions: []
+
+  client: null
+  socket: null
+
+  headers: {
+    login: SyncConfig.api_key,
+    passcode: SyncConfig.auth_token
+  }
+
+  available: ->
+    !!window.Stomp
+
+  connect: ->
+    @socket = new window.SockJS(SyncConfig.websocket)
+    @client = window.Stomp.over(@socket)
+
+    # SockJS does not support heart-beat: disable heart-beats
+    @client.heartbeat.outgoing = 0
+    @client.heartbeat.incoming = 0
+    @client.debug = Sync.onDebug if SyncConfig.debug_flag
```

Comment:
Is there something about STOMP that makes us need to enable debugging? Otherwise, let's leave it out until we refactor this javascript.